### DISCUSSION TITLE
Should a page deleted through visual editor still be available in controllers code?

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#163] [Discuss: New Feature Ideas]
I deleted a page through visual editor. However, it is still available at controllers/routes. 

Should it be user's responsibility to manually delete it?

**Comment by cruxcode** [comment]
I think we should not delete the files automatically because the user might want to reference or copy some code from these controllers' file.

**Reply by darshitac11** [reply]
Sounds good!
